### PR TITLE
Limbus

### PR DESCRIPTION
**Pull request by Brierre**
Redo of Limbus files so doors open and chests pop

### CODE REVIEW

**Comment by wrenffxi** [comment]
i'm workin' with her to fix the CI bonk

File: scripts/zones/Apollyon/mobs/Fossil_Quadav.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+require("scripts/globals/limbus");
+mixins = {require("scripts/mixins/job_special")};
+-----------------------------------
+
+function onMobFight(mob,target)
+end;
+
+function onMobDeath(mob, player, isKiller)
+    local helpers = GetServerVariable("deadQuadHelpers");
+    SetServerVariable("deadQuadHelpers",helpers +1);
```
Comment (Hozu) [inline scripts/zones/Apollyon/mobs/Fossil_Quadav.lua]: onMobDeath runs once for every party member. I don't think that's what you want to happen.

**Comment by Brierre** [comment]
Thank you, fixed. Lmk if there's anything else screwy!

**Comment by TeoTwawki** [comment]
could this be rebased onto the bcnm branch? because conflicts when we merge bcnm branch with master.

the team would really appreciate it if all changes to limbus dynamis bcnm enm etc could all be directed there until it gets merged.

**Comment by Brierre** [comment]
Sure, it can be. I just knew that branch has been in the works for months.

**Comment by Brierre** [comment]
Haha, for now.  I figured by the time I get around to getting a rebase done
I might find other things to change.

**Pull request closed without merging**